File: WingedPlate.Domain/Entities/CommentEntity.cs
namespace WingedPlate.Domain.Entities;

public class CommentEntity : EntityBase
{
    public string Content { get; set; } = string.Empty;

    public int UserId { get; set; } = 0;

    public UserEntity User { get; set; } = new();

    public int RecipeId { get; set; } = 0;

    public RecipeEntity Recipe { get; set; } = new();
}

File: WingedPlate/src/entities/comment.entity.ts
import { RecipeEntity } from "./recipe.entity";
import { UserEntity } from "./user.entity";

export class CommentEntity
{
    public content: string = "";

    public userId: number = 0;

    public user: UserEntity = new UserEntity();

    public recipeId: number = 0;

    public recipe: RecipeEntity = new(RecipeEntity);
}